src/config.ts

```diff
@@ -1,7 +1,13 @@
 import { MessageEmbed } from 'discord.js';
 
 export default {
-  COMMAND_PREFIX: '!',
+  COMMAND_PREFIX: '$',
+  INTRO_CHANNEL: 'introductions',
+  GENERAL_CHANNEL: 'general',
+  REACTIONS_COUNT: 5,
+  ROLE: {
+    HIGH_VALUE: 'high value'
```

Comment:
Another interesting refactor, I think we can use this for more conditions in the future 👍